src/HotChocolate/Core/test/Types.Tests/Types/Descriptors/Conventions/XmlDocumentationProviderTests.cs

```diff
@@ -95,6 +95,22 @@ public void When_type_has_description_then_it_it_resolved()
             Assert.Equal("I am the most base class.", description);
         }
 
+        [Fact]
+        public void When_we_use_custom_documentation_files_they_are_correctly_loaded()
+        {
+            // arrange
+            var documentationProvider = new XmlDocumentationProvider(
+                new XmlDocumentationFileResolver(_ => "Dummy.xml"),
+                new NoOpStringBuilderPool());
+
+            // act
+            var description = documentationProvider.GetDescription(
+                typeof(BaseBaseClass));
+
+            // assert
+            Assert.Equal("I am the most base class from dummy.", description);
```

Comment:
We check that we get the description from our modified config instead of the standard config. The standard config would state `I am the most base class.`. 

With this small integration test we have proved that our option really would work. 

The original test `When_type_has_description_then_it_it_resolved ` proves that the old behavior is still intact.